### PR TITLE
SWAT-1372: Use entityqueue to order case studies on work page.

### PR DESCRIPTION
**Pull request by antoniotag**

### CODE REVIEW

**Comment by antoniotag** [comment]
This needs to be merged after https://github.com/thirdandgrove/third-and-grove-cms/pull/23 is merged and deployed to production.

**Comment by grantglidewell** [comment]
What do we need to do for this to get back on track? @antoniotag

**Comment by antoniotag** [comment]
@grantglidewell So this depends on an entity queue that will be created on the CMS side after the PR for it is merged on that repo: https://github.com/thirdandgrove/third-and-grove-cms/pull/23. After that is merged, this code will read that queue through the API and display the ordered case studies.

File: src/pages/work.js [hunk 1]
```diff
@@ -182,7 +189,7 @@ export default () => {
       }
     }
   `);
-  const studies = allCaseStudy.nodes;
+  const studies = allEntitySubqueueCaseStudies.nodes[0].relationships.items;
```
[the remark this answers is not in the document]
Comment (antoniotag) [reply src/pages/work.js]: @grantglidewell I've limited the query to just one node.

**Comment by grantglidewell** [comment]
@antoniotag this build is failing in netlify and gatsby, does this work for you locally building against the prod back end? it looks like the changes required in Drupal were merged but may not be available?

**Pull request merged**